public/index.html

```diff
@@ -2,27 +2,28 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/car-logo.jpeg" />
-    <!-- <link rel="apple-touch-icon" href="../src/Assets/car-logo.jpeg" /> -->
+    <link rel="icon" href="%PUBLIC_URL%/logo.svg" />
+
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
-    <!--
-      manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
-    -->
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
This is logo192 so essentially with this line of code just for apple users, in some case they would see the react laog, while other people would continue to see the regular xtreme cars logo
![logo192](https://user-images.githubusercontent.com/104024981/165922819-fd429e28-8df5-402d-b8a9-3d4b1daec9f7.png)